.ci/pitest-suppressions/pitest-sizes-suppressions.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<suppressedMutations>
+  <mutation unstable="false">
+    <sourceFile>ExecutableStatementCountCheck.java</sourceFile>
+    <mutatedClass>com.puppycrawl.tools.checkstyle.checks.sizes.ExecutableStatementCountCheck</mutatedClass>
+    <mutatedMethod>visitSlist</mutatedMethod>
+    <mutator>org.pitest.mutationtest.engine.gregor.mutators.experimental.NakedReceiverMutator</mutator>
+    <description>replaced call to com/puppycrawl/tools/checkstyle/api/DetailAST::getParent with receiver</description>
+    <lineContent>DetailAST parent = ast.getParent();</lineContent>
+  </mutation>
+
+  <mutation unstable="false">
+    <sourceFile>ExecutableStatementCountCheck.java</sourceFile>
+    <mutatedClass>com.puppycrawl.tools.checkstyle.checks.sizes.ExecutableStatementCountCheck$Context</mutatedClass>
+    <mutatedMethod>&lt;init&gt;</mutatedMethod>
+    <mutator>org.pitest.mutationtest.engine.gregor.mutators.experimental.MemberVariableMutator</mutator>
+    <description>Removed assignment to member variable count</description>
+    <lineContent>count = 0;</lineContent>
+  </mutation>
+
+  <mutation unstable="false">
+    <sourceFile>FileLengthCheck.java</sourceFile>
+    <mutatedClass>com.puppycrawl.tools.checkstyle.checks.sizes.FileLengthCheck</mutatedClass>
+    <mutatedMethod>&lt;init&gt;</mutatedMethod>
+    <mutator>org.pitest.mutationtest.engine.gregor.mutators.experimental.MemberVariableMutator</mutator>
+    <description>Removed assignment to member variable max</description>
+    <lineContent>private int max = DEFAULT_MAX_LINES;</lineContent>
+  </mutation>
+
+  <mutation unstable="false">
+    <sourceFile>MethodLengthCheck.java</sourceFile>
+    <mutatedClass>com.puppycrawl.tools.checkstyle.checks.sizes.MethodLengthCheck</mutatedClass>
+    <mutatedMethod>countUsedLines</mutatedMethod>
+    <mutator>org.pitest.mutationtest.engine.gregor.mutators.NonVoidMethodCallMutator</mutator>
+    <description>removed call to com/puppycrawl/tools/checkstyle/api/DetailAST::getLineNo</description>
+    <lineContent>final int startLineNo = ast.getLineNo();</lineContent>
+  </mutation>
```

Comment:
@Vyom-Yadav please open PR with hardcoded mutation for this.

Reply:
#11998, I won't be adding comment explanation comment at #11719, this particular mutant is very well documented at https://pitest.org/quickstart/mutators/#NON_VOID_METHOD_CALLS